source/process/community-process.rst

```diff
@@ -61,7 +61,15 @@ The various community channels are outlined below, along with the responsibiliti
 
     - Ops Team @mentions the Community PM with a list that includes issues:
 
-      - where status requires updating (Under Review, Planned, Started, etc.)
+      - where status requires updating
+      
+        - Under Review: Before a ticket is made (spec is needed)
```

Comment:
Maybe after a ticket is made for it, but spec isn't completed.

That way there's a link community can follow